stubs/tifftools/constants.pyi

```diff
@@ -28,6 +28,7 @@ class TiffConstantSet(Generic[_TiffConstantT]):
     ) -> _TiffConstantT | None: ...
 
 class TiffTag(TiffConstant):
+    altnames: set[str] | None
```

Comment:
See https://github.com/DigitalSlideArchive/tifftools/blob/0c39e95119c302f553d818260a28cbd83a7da98e/tifftools/constants.py#L627 as an example of a tag with multiple names.